common/app/views/fragments/meta/metaInline.scala.html

```diff
@@ -32,7 +32,9 @@
                   <a class="content__section-label__link"
                       data-link-name="article section"
                       href="@LinkTo {/@item.content.sectionLabelLink}">
+                      <span>
```

Comment:
Should this span and the one under `content__series-label__link` should have a class name so you can target both easily?